index.html

```diff
@@ -113,6 +140,35 @@ <h3 id="neural-network-inference">
       retrieve the output when hardware execution completes.
       </li>
     </ul>
+    <h3 id="task-specific-apis">
+      Task-specific APIs
+    </h3>
+    <p>
+      A set of higher-level APIs that:
+    </p>
+    <ul>
+      <li>Detect the language of the given input text;</li>
+      <li>Translate input text to other languages;</li>
+      <li>Produce textual summaries of input text;</li>
+      <li>Produce new textual material given a writing task;</li>
+      <li>Rephrase input text.</li>
+    </ul>
+    <h3 id="prompt-api">
+      Prompt API
+    </h3>
+    <p>
+      A general-purpose API to prompt a language model directly using common
+      promping techniques to accomplish a variety of tasks, including:
+    </p>
+    <ul>
+      <li>Classification, tagging, and keyword extraction of arbitrary text;</li>
+      <li>Helping users compose text, such as blog posts, reviews, or biographies;</li>
+      <li>Summarizing, e.g. of articles, user reviews, or chat logs;</li>
+      <li>Generating titles or headlines from article contents;</li>
+      <li>Answering questions based on the unstructured contents of a web page;</li>
+      <li>Translation between languages;</li>
+      <li>Proofreading.</li>
+    </ul>
     <p>
```

Comment:
Nit: these paragraphs are now under "prompt API". Maybe they should move up to the top?